docs/UserGuide.adoc

```diff
@@ -58,16 +58,21 @@ Format: `help`
 === Adding a person: `add`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
+Format: `add n/NAME m/MATRIC_NUMBER g/GENDER p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG] [tt/NUM_OF_MONTHS/TEMPORARY TAG]...` +
```

Comment:
Consider whether matric number and gender should be compulsory tags?

Reply:
it's a good point. I think gender is compulsory but not matric no. :)

Reply:
oh sry...I accidentally deleted ur ReviewMsg...is there any way to recover it?